features/fixtures/app/gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
```

Comment:
@fractalwrench I'm assuming this shouldn't be checked in?

Reply:
It's fine to check this in generally in JVM projects. The mazerunner tests update this file if the value of `GRADLE_WRAPPER_VERSION` is set however, which can mean files in the working tree change when the tests run.